package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fury-adapter-swagger",
-  "version": "0.8.0-pre.0",
+  "version": "0.8.0-pre.2",
```

Comment:
Since you've changed the version number, should CHANGELOG be updated to reflect this too?